File: java/angka.java
import java.util.Scanner;
public class angka {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Masukkan angka pertama: ");
        int a = sc.nextInt();
        System.out.print("Masukkan angka kedua: ");
        int b = sc.nextInt();

        if (a > b) {
            System.out.println(a + " : Angka pertama lebih besar dari angka kedua : " + b);
        } else if (a < b) {
            System.out.println(a + " : Angka pertama lebih kecil dari angka kedua : " + b);
        } else {
            System.out.println(" : Kedua angka sama : ");
        }
        System.out.println("Apakah Keduanya Sama? " + (a == b));
    }
}

File: c++/angka.cpp
#include <conio.h>
#include <iostream>
#include <stdio.h>

using namespace std;

main()
{
    int a, b;
    cout << "Masukkan angka pertama : ";
    cin >> a;
    cout << "Masukkan angka kedua : ";
    cin >> b;

    if (a > b) {
        cout<< a << " : Lebih Besar dari : "<< b << endl;
    } else if (a < b) {
        cout<< b << " : Lebih Kecil dari : "<< b << endl;
    } else {
        cout<< a << " Keduanya sama besar " << b << endl;
    }

    cout << "Apakah Keduanya Sama Besar? "<< (a == b) << endl;
}